make/photon/prepare/migrations/version_1_10_0/harbor.yml.jinja

```diff
@@ -287,7 +287,6 @@ external_redis:
   # db_index 0 is for core, it's unchangeable
   registry_db_index: {{ external_redis.registry_db_index }}
   jobservice_db_index: {{ external_redis.jobservice_db_index }}
-  chartmuseum_db_index: {{ external_redis.chartmuseum_db_index }}
```

Comment:
Do we need to remove the chartmuseum related config in the old version template? Not sure about that.

Reply:
+1, they are history file, no need to change it.